tests/links_tests/connection_tests/test_conv_2d_activ.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+
+import numpy as np
+
+import chainer
+from chainer import cuda
+from chainer import testing
+from chainer.testing import attr
+
+from chainercv.links import Conv2DActiv
+
+
+def _add_one(x):
+    return x + 1
+
+
+@testing.parameterize(
+    {'args_style': 'explicit'},
+    {'args_style': 'None'},
+    {'args_style': 'omit'}
+)
+class TestConv2DActivForward(unittest.TestCase):
```

Comment:
Can you merge `TestConv2DActivForward` and `TestConv2DActivBarckward` into one class?